### PR TITLE
Splitter: fix AXE scrollable accessibility issue

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**
**Description:**
The main problem is there is axe scrollable accessibility issue. Issue exist because item content has no focusable content inside of it thats why it make keyboard navigation not have ability to scroll inside.

**Solution:**
I applied added attribute tabindex in pane-content to make the whole content focusable allowing user the ability to scroll instead of adding a focusable item (e.g a focusable button) in the pane which it doesnt follow our original prototype design.

### CODE REVIEW

File: apps/demos/Demos/Splitter/Overview/React/PaneContent.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const getFilteredDimensionOptions = (data) => Object.entries(data)
   .map(([key, value]) => ({ key, value }));
 
 const PaneContent = (data) => (
-  <div className="pane-content">
+  <div className="pane-content" tabIndex={0}>
```
Comment (ksercs) [inline apps/demos/Demos/Splitter/Overview/React/PaneContent.tsx:20]: and what about demo for ReactJS?

**Review by ksercs — DISMISSED** [review]

File: apps/demos/Demos/Splitter/Overview/React/PaneContent.tsx (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Splitter/Overview/React/PaneContent.tsx:20]: [72a19e6](https://github.com/DevExpress/DevExtreme/pull/27209/commits/72a19e6b474cc978034926f838f491d76f49a52e)

**Review by ksercs — DISMISSED** [review]

File: apps/demos/testing/common.test.js [hunk 2]
```diff
@@ -138,6 +138,7 @@ const SKIPPED_TESTS = {
   React: {
     Charts: [
       { demo: 'PiesWithEqualSize', themes: [THEME.material] },
+      { demo: 'CustomAnnotations', themes: [THEME.material] },
```
Comment (ksercs) [inline apps/demos/testing/common.test.js:141]: Seems like only vectorMap test should be skipped, no? 
Do not make excess skips please

Comment (ksercs) [reply apps/demos/testing/common.test.js:141]: Okay I see this test is unstable as well

Comment (nikkithelegendarypokemonster) [reply apps/demos/testing/common.test.js:141]: the error exists on Charts so its necessary to include it on Charts

Comment (nikkithelegendarypokemonster) [reply apps/demos/testing/common.test.js:141]: The test error exists in Charts so its necessary to add the skipped test there instead of Vector Map

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**